WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductVariationFormViewModel.swift

```diff
@@ -26,6 +26,12 @@ final class ProductVariationFormViewModel: ProductFormViewModelProtocol {
         isUpdateEnabledSubject
     }
 
+    /// Emits a void value informing when there is a new variation price state available
+    var newVariationsPrice: Observable<Void> {
+        // No-op
+        PublishSubject<Void>()
+    }
```

Comment:
Looks like this can also just be 

```suggestion
    var newVariationsPrice: Observable<Void> = PublishSubject<Void>()
```

I don't think we'll save a lot of performance if we create it lazily.